impl/src/main/java/org/arquillian/extension/governor/impl/ClosePassedDeciderImpl.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * JBoss, Home of Professional Open Source
+ * Copyright 2015, Red Hat, Inc. and/or its affiliates, and individual
+ * contributors by the @authors tag. See the copyright.txt in the
+ * distribution for a full listing of individual contributors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.arquillian.extension.governor.impl;
+
+import org.arquillian.extension.governor.api.ClosePassedDecider;
+
+import java.lang.annotation.Annotation;
+import java.lang.reflect.Method;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * @author <a href="mailto:mbasovni@redhat.com">Martin Basovnik</a>
+ */
+public class ClosePassedDeciderImpl implements ClosePassedDecider {
+
+    private static final Logger logger = Logger.getLogger(ClosePassedDeciderImpl.class.getName());
+
+    private Map<Annotation, Boolean> closableAnnotationMap = new HashMap<Annotation, Boolean>();
+
+    @Override
+    public Map<Annotation, Boolean> get() {
+        return closableAnnotationMap;
+    }
+
+    @Override
+    public void setClosable(Annotation annotation, boolean closeable) {
+        for (Map.Entry<Annotation, Boolean> entry : closableAnnotationMap.entrySet()) {
+            Annotation oldAnnotation = entry.getKey();
+            if (oldAnnotation.annotationType().equals(annotation.annotationType())) {
+                String oldId = getAnnotationValue(oldAnnotation);
+                String id = getAnnotationValue(annotation);
+                if (oldId != null && oldId.equals(id)) {
+                    closableAnnotationMap.put(oldAnnotation, entry.getValue().booleanValue() & closeable);
```

Comment:
No. `@Jira` annotation has two properties `value` and `force`. Image situation that you set property `force` in `arquillian.xml` to `true` and then you have two `@Jira` annotations and only one of them contains property `force` set to true. It means that both of them will be always executed, but `equals` and `hashCode` methods will not evaluate those annotations to be the same. So `Map.get()` will not work. Therefore I compare annotations only according to its value. Both tests must succeed to close the issue. Therefore I use binary and `&` operator.

Example:
```java
@RunWith(Arquillian.class)
public class DummyTest
{
    @Test
    @Jira(value = "DUMMY-1")
    public void test1() {
        Assert.assertTrue(true);
    }

    @Test
    @Jira(value = "DUMMY-1", force="true")
    public void test2() {
        Assert.assertTrue(true);
    }
}
```
```xml
<arquillian>
    <extension qualifier="governor-jira">
        <property name="force">true</property>
    </extension>
</arquillian>
```